__init__.py

```diff
@@ -21,6 +21,9 @@
  ***************************************************************************/
  This script initializes the plugin, making it known to QGIS.
 """
+import sys, os
+
+sys.path.append(os.path.join(os.path.realpath(__file__),os.pardir))
```

Comment:
Dit vind ik er gevaarlijk uitzien. Als je de plugin checkout hebt gesymlinkt dan krijg je een directory op het path die niet echt de bedoeling is. 

(Los daarvan hoort er een spatie achter de komma te staan :-) )